addons/account/models/account_analytic_line.py

```diff
@@ -110,14 +151,20 @@ class AccountAnalyticLine(models.Model):
     _description = 'Analytic Line'
 
     product_id = fields.Many2one('product.product', string='Product', check_company=True)
-    general_account_id = fields.Many2one('account.account', string='Financial Account', ondelete='restrict', readonly=True,
-                                         related='move_id.account_id', store=True, domain="[('deprecated', '=', False), ('company_id', '=', company_id)]",
-                                         compute_sudo=True)
+    general_account_id = fields.Many2one('account.account', string='Financial Account', ondelete='restrict', readonly=False,
+                                         domain="[('deprecated', '=', False), ('company_id', '=', company_id)]",
+                                         compute="_compute_general_account_id")
```

Comment:
not stored?

also put the compute, readonly (and store?) arg on the same line, as they're linked and whenever someone changes the value of one of them the other shouldn't be disregarded